packages/product/src/services/product-collection.ts

```diff
@@ -1,7 +1,8 @@
-import { ProductCollection } from "@models"
 import { Context, DAL, FindConfig, ProductTypes } from "@medusajs/types"
 import { ModulesSdkUtils, retrieveEntity } from "@medusajs/utils"
 
+import { ProductCollection } from "@models"
```

Comment:
question: when did we start supporting this type of import? must have missed it

Reply:
The product modules is using these paths for models, services and repositories.
https://github.com/medusajs/medusa/blob/develop/packages/product/tsconfig.json#L22

Reply:
Thanks, I like it!